gui/system/migrations/0007_auto_201708211858

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-08-21 18:58
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def change_boot_scrubs(apps, schema_editor):
+    boot_scrub = apps.get_model('system', 'Advanced')
```

Comment:
Just for the sake of consistency, lets change that to
Advanced = apps.get_model('system', 'Advanced')
queryset = Advanced.objects.filter(adv_boot_scrub=35)